Xpirit-Vsts-Release-Terraform/task.json

```diff
@@ -85,6 +85,18 @@
       "helpMarkDown": "Select the Azure Resource Manager subscription.",
       "visibleRule": "ManageState == true"
     },
+    {
+      "name": "StorageAccountResourceGroup",
+      "type": "pickList",
```

Comment:
One thing we weren't clear on is how to wire up to get it to load the list of available resource groups the same way the storage account list populates. I assume it's based off the `dataSourceBindings` but I wasn't clear yet on how that works.

Reply:
Thx, I'll make a test version as soon I have time. The code looks good! I'll have a look at bindings. I think I have done or seen that somewhere

Reply:
Great, thanks!